### DISCUSSION TITLE
Are the tracking hosts and ad hosts on a different list or are they combined?

### DISCUSSION THREAD

**Discussion by kennbr34** [Trellmor/bind-adblock#37] [General]
I really am not that concerned with blocking tracking hosts, and it's become an annoyance because so many links will not work since they're facilitated through some kind of tracking service. I figure I could remove the disconnect.me tracking list, but what about Peter Lowe's list? That seems to imply that it contains both tracking and ad hosts.

I know I can just add certain hosts to the whitelist, but editing a whitelist and adding a host every time I can't follow a link seems like it will get annoyingly cumbersome. I'd prefer to just not block tracking hosts.

**Comment by Trellmor** [comment]
Peter Lowe's list has an additional parameter [notrackers](https://pgl.yoyo.org/as/formats.php#notrackers): https://pgl.yoyo.org/as/serverlist.php?hostformat=nohtml&showintro=0&notrackers=1
You can edit the config and modify the url there.

But [StevenBlack/hosts](https://github.com/StevenBlack/hosts) also includes Peter Lowe's list with trackers, so you probably need to remove this list if you want to get rid of tracker hostnames.
